Examples/Delegates.playground/Contents.swift

```diff
@@ -0,0 +1,117 @@
+import Foundation
+
+// MARK: - Public optional delegate example
+
+/// A delegate protocol, which contains events of Actor1 class
+protocol Actor1Delegate: AnyObject {
+
+    /// Event action without any additional parameters
+    func actor1DidSomething(_ actor: Actor1)
+
+    /// Event action with additional parameters
+    func actor1(_ actor: Actor1, didSomethingWithValue value: Int)
+}
+
+/// Actor class, which produces some actions
+class Actor1: NSObject {
```

Comment:
Let's choose names that are close to reality, for example `ListView`, `func listViewDidTapDone(_ view: ListView)`, etc.